src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -1087,68 +1087,45 @@ internal virtual void FillRequestStream(HttpRequestMessage request)
 
             if (Form is not null)
             {
-                // Content headers will be set by MultipartFormDataContent which will throw unless we clear them first
-                WebSession.ContentHeaders.Clear();
-
                 var formData = new MultipartFormDataContent();
                 foreach (DictionaryEntry formEntry in Form)
                 {
                     // AddMultipartContent will handle PSObject unwrapping, Object type determination and enumerateing top level IEnumerables.
                     AddMultipartContent(fieldName: formEntry.Key, fieldValue: formEntry.Value, formData: formData, enumerate: true);
                 }
 
-                SetRequestContent(request, formData);
+                request.Content = SetRequestContent(request, formData);
             }
-            // coerce body into a usable form
+            // Coerce body into a usable form
```

Comment:
Such comments reported by CodeFactor. It is better to move to line1102. Below too.